internal/listener/listener.go

```diff
@@ -86,16 +85,18 @@ func (l *Listener) ProcessEvent(w http.ResponseWriter, req *http.Request) {
 		}
 	}
 
-	obj, err := object.FromTags(l.db, ev.Tags)
+	ctx := req.Context()
```

Comment:
Please explain the idea behind using `req.Context()` here.

I'm not sure if we want this in the end, but that will probably depend on how we handle HA in the end.

Typically, that context is used to stop generating a response soon after a connection has died because this means that the response won't be received anyways. But in our use-case, the main effect of the request is tracking the incident state and sending notifications, so if we've received the request, it can make sense to continue processing it even after the connection that sent the event died.